ui/apps/platform/src/Containers/NetworkGraph/simulation/NetworkPoliciesYAML.tsx

```diff
@@ -53,17 +63,15 @@ function NetworkPoliciesYAML({ yaml, generateNetworkPolicies }: NetworkPoliciesY
         />
     ) : null;
 
-    if (!yaml || yaml === '') {
-        return (
-            <Bullseye>
-                <EmptyState variant={EmptyStateVariant.xs}>
-                    <Title headingLevel="h4" size="md">
-                        No network policies
-                    </Title>
-                </EmptyState>
-            </Bullseye>
-        );
-    }
+    const revertRecentYAML = undoNetworkPolicies ? (
+        <CodeEditorControl
+            icon={<UndoIcon />}
+            aria-label="Revert most recently applied YAML"
+            toolTipText="Revert most recently applied YAML"
```

Comment:
nit: If these are the same, perhaps we should define a string constant and reuse it in both props.